platform-manageability-agent/cmd/platform-manageability-agent/platform-manageability-agent.go

```diff
@@ -266,6 +285,31 @@ func main() {
 	log.Infof("Platform Manageability Agent finished")
 }
 
+func enableService(action, service string) error {
+	allowedActions := map[string]bool{"unmask": true, "enable": true, "start": true}
+	allowedServices := map[string]bool{"lms.service": true}
+
+	if !allowedActions[action] || !allowedServices[service] {
```

Comment:
Since this is only used for the lms.service, there is no need to provide the service name as an input, we can just provide the required systemctl action to perform on the service and check that is a valid one